air/tests/test_module/negative_tests/execution_step.rs

```diff
@@ -0,0 +1,435 @@
+/*
+ * Copyright 2023 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+use air::{CatchableError, LambdaError};
+use air_test_utils::prelude::*;
+
+#[test]
+fn local_service_error() {
+    let client_peer_id = "some_peer_id";
+    let script = f!(r#"
+        (call "{client_peer_id}" ("m" "f") ["arg1"] other)
+    "#);
+
+    let mut client_vm = create_avm(echo_call_service(), client_peer_id);
+    let result = client_vm
+        .runner
+        .call(&script, "", "", client_peer_id, 0, 0, None, <_>::default())
+        .unwrap();
+
+    let err_msg = "some error".to_string();
+    let call_service_result = air_test_utils::CallServiceResult::err(10000, json!(err_msg));
+    let call_results_4_call = maplit::hashmap!(
+        1 => call_service_result,
+    );
+    let result = client_vm
+        .runner
+        .call(script, "", result.data, client_peer_id, 0, 0, None, call_results_4_call)
+        .unwrap();
+
+    let err_msg = std::rc::Rc::new(f!("\"{err_msg}\""));
+    let expected_error = CatchableError::LocalServiceError(10000, err_msg);
+    assert!(check_error(&result, expected_error));
+}
+
+#[test]
+fn ap_scalar_variable_not_found_error() {
```

Comment:
Suggest naming tests in this file following this pattern
`<error_name>_<specific>`, so this one would be `variable_not_found_ap_scalar`, and without `error`.

Reply:
Done